e2e/steps_file.js

```diff
@@ -59,11 +59,10 @@ module.exports = function () {
       }
     },
 
-    seeCheckAnswers(checkAnswerTitle) {
-      this.click('Continue');
-      this.waitForElement('.check-your-answers');
+    async seeCheckAnswersAndCompleteEvent(checkAnswerTitle = 'Check the information below carefully.') {
```

Comment:
Is this likely to change? Do we need as a param?

Reply:
will remove - no need for param as will not change